wagtailcaptcha/forms.py

```diff
@@ -1,6 +1,6 @@
 from __future__ import absolute_import, unicode_literals
 
-from captcha.fields import ReCaptchaField
+from django_captcha.fields import ReCaptchaField
```

Comment:
Isn't this supposed to be `django_recaptcha`?

Reply:
Yes, it looks like it. There is [no reference](https://github.com/search?q=repo%3Adjango-recaptcha%2Fdjango-recaptcha%20django_captcha&type=code) to `django_captcha` in django-recaptcha.